docs/features/software-catalog/installation.md

```diff
@@ -21,10 +22,6 @@ cd packages/app
 yarn add @backstage/plugin-catalog
 ```
 
-Make sure the version of `@backstage/plugin-catalog` matches the version of
```

Comment:
This isn't true any more, right? Packages have different versions now.

Reply:
Aah yes!